### ISSUE TITLE
Handle last window closing

### ISSUE THREAD

**Issue by saagarjha** [saagarjha/MacCast#8]
When the last window closes, we should throw up the connection screen again when the app is launched. It is probably good enough to kill the app/discard all scenes and let the system recreate it for us

**Comment by kgelnerFW** [comment]
Heads up that related to this, I killed off the last Ensemble window in AVP, then killed off the app on the mac (simulator) and the Mac app crashed :

<img width="915" alt="Screenshot 2024-02-06 at 3 15 47 PM" src="https://github.com/saagarjha/Ensemble/assets/115652506/46da16ad-b82e-4fe3-885a-cee039389d41">

I didn't open a new issue as I thought the problem might be addressed by this open issue.

**Comment by saagarjha** [comment]
Yep I know about that one, I don't handle errors or connection changes at all right now. I'll get around to it eventually!
